### ISSUE TITLE
Duplicated quotes on string conversion

### ISSUE THREAD

**Issue by acmb** [eldoy/csvstrom#1]
Currently, `csvstrom` seems to duplicate quotes when converting CSV file values.

For instance, if our input is a CSV file as below:
```csv
id,name
"123","navn"
```
The resulting JSON file will be:
```json
[
   {
      "id": "\"123\"",
      "name": "\"navn\""
   }
]
```
The ***expected*** JSON file would be:
```json
[
   {
      "id": "123",
      "name": "navn"
   }
]
```
We should probably identify CSV values in-between `"` or `'` as strings, but remove quotes before the actual conversion.
We still need to be careful not to convert `"123"` *(string)* to `123` *(number)*.

**Comment by eldoy** [comment]
The conversion must adhere to the spec here: https://en.wikipedia.org/wiki/Comma-separated_values

We probably need a converter function to handle that? Can be tested separately.

**Comment by eldoy** [comment]
Fixed in `0.2.0`
